content/en/faq/security/verify-mtls-encryption.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How can I verify Istio mTLS indeed encrypt the traffic?
+weight: 40
+---
+
+Once you turned on the mutual TLS, traffic is encrypted. We will show that through `tcpdump` in the
+sidecar container.
+
+To enable tcpdump, you need to install Istio with proxy priviledged enabled.
+
+{{< text bash >}}
+$ istioctl manifest apply --set values.global.proxy.privileged=true
+{{< /text >}}
+
+{{< warning >}}
+This installation is only for demo purpose, please revert the change after you are done for better security!
+{{< /warning>}}
+
+Deploy httpbin, sleep workloads, and a sleep-legacy deployment which does not have sidecar injected.
```

Comment:
I would refer to the authentication task instead. IMO, the faq should be short, and skip demo set up. Otherwise, you can make this become a task by itself (we used to have mtls deepdive that show how to debug/verify mTLS, though it is deleted recently)

Reply:
This also doesn't really sound like an answer to the general question: `How can I verify Istio mTLS indeed encrypt the traffic?` In a production env, users can't do this, so this is more of a simple demo task, not a general answer to a frequently asked question. Also, do we really need a task to demonstrate how to use tcpdump, or would a tip in the tls migration doc be good enough (for example, `you can verify that traffic is encrypted by running tcpdump in a proxy container, but only in an Istio installation with values.global.proxy.privileged=true`)?

Reply:
Some of our FAQ page does include the some cli command, though i agree this is a bit longer than others.

i'm fine to move the info to a task page. but would it be appropriate  to put a link in the FAQ page?

I just have seen many many users are not so sure about our mTLS effect. They feel, I know request can still pass, you take care of mtls, but how can i know learn more?

Reply:
moved to the migration doc.